src/foam/nanos/pm/PMInfo.js

```diff
@@ -80,6 +80,7 @@ foam.CLASS({
 
   methods: [
     {
+      synchronized: true,
```

Comment:
What issue are you seeing? synchronization in PMs is the last thing we need as they need to be super fast.